package.json

```diff
@@ -14,7 +14,7 @@
     "build:dir": "node .electron-vue/build.js && electron-builder --dir",
     "build:clean": "cross-env BUILD_TARGET=clean node .electron-vue/build.js",
     "build:dev": "node .electron-vue/build.js",
-    "dev": "node .electron-vue/dev-runner.js",
+    "dev": "cross-env MARKTEXT_ERROR_INTERACTION=0 node .electron-vue/dev-runner.js",
```

Comment:
I prefer that you set `MARKTEXT_ERROR_INTERACTION` in your environment variables.

Reply:
Ok!